src/components/circular-progress/circular-progress.scss

```diff
@@ -76,6 +76,18 @@
         box-shadow: var(--button-shadow-pressed);
     }
 }
+.prefix-value {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.prefix {
+    color: var(--circular-progress-fill-color, var(--mdc-theme-primary));
```

Comment:
I don't think we should use either of these colors. 
It's better to give prefix and suffix their own CSS props if they should be styled.